core/5.0/debug.d.ts

```diff
@@ -0,0 +1,183 @@
+// Based on https://www.lua.org/manual/5.3/manual.html#6.10
```

Comment:
5.0 declarations are based on the 5.3 reference?

Reply:
Certainly not! Fixed with [706f79b](https://github.com/TypeScriptToLua/lua-types/pull/59/commits/706f79bb3aaa6f3e9575dc3140cee5841eb280bb).